students/swietczak_monika/lesson_08_regular_expressions/email_validator.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+regex = re.compile(r"[^@]+@[^@]+\.[^@]+")
+
+
+def email_validator(some_email):
+    if regex.match(some_email) is None:
+        print("email is not valid")
+    else:
+        print("email is valid")
+
+
+# email_validator("mail123@test.com")
+# email_validator("mail.mail")
+# email_validator("mail@test")
```

Comment:
Please consider to add py.test for verification.

There is available perfect feature (parametrize) for calling the same method many times.

```
class TestEmailAddressValidator:
    @pytest.mark.parametrize("correct_address", [
        'aaa@aaa.pl',
        'a-a-a@aaa.pl',
        'a12a@12a.com',
    ])
    def test_valid_email_address(self, correct_address):
        assert email_validator(correct_address)

    @pytest.mark.parametrize("incorrect_address", [
        'aaa@a@aa.pl',
        'aaa@@aa.pl',
        'a12a.com',
    ])
    @pytest.mark.xfail
    def test_incorrect_email_address(self, incorrect_address):
        assert email_validator(incorrect_address)
```